pkg/cmd/printer/broadcast.go

```diff
@@ -0,0 +1,58 @@
+package printer
+
+import (
+	"context"
+
+	"github.com/aquasecurity/tracee/pkg/logger"
+	"github.com/aquasecurity/tracee/types/trace"
+)
+
+// Broadcast is a printer that broadcasts events to multiple printers
+type Broadcast struct {
+	eventsChan []chan trace.Event
+}
+
+// NewBroadcast creates a new Broadcast printer
+func NewBroadcast(ctx context.Context, printers []EventPrinter) *Broadcast {
+	eventsChan := make([]chan trace.Event, 0, len(printers))
+
+	for _, printer := range printers {
+		// we use a buffered channel to avoid blocking the event channel,
+		// we match the size of ChanEvents buffer
+		eventChan := make(chan trace.Event, 1000)
+		eventsChan = append(eventsChan, eventChan)
+
+		go startPrinter(ctx, eventChan, printer)
+	}
+
+	return &Broadcast{eventsChan: eventsChan}
+}
+
+// Print broadcasts the event to all printers
+func (b *Broadcast) Print(event trace.Event) {
+	for _, c := range b.eventsChan {
+		// we use select to avoid blocking the print loop if one of the printers is slow
+		// in case the event can't be sent because the buffered channel is full, we drop it
+		// and log an warning
+		select {
+		case c <- event:
+		default:
+			// TODO: add metrics about not printed events
+			logger.Debug("dropping event due to slow printer", "event", event)
```

Comment:
I think this deserves at least info level log message

Reply:
Info is default level, so depending on the volume of events it will output more logs than events.

Reply:
I think this should happen too frequently (at least I hope so), and if something is dropped here, there should be a way for the user to be aware of that. So as long as we don't have metrics for events dropped here, we should expose this information somehow